package/yast2-security.spec

```diff
@@ -27,19 +27,20 @@ Group:          System/YaST
 License:        GPL-2.0
 BuildRequires:  doxygen pkg-config perl-XML-Writer update-desktop-files yast2-pam
 BuildRequires:  yast2-devtools >= 3.1.10
-BuildRequires:  yast2-mail
 BuildRequires:  rubygem(yast-rake) >= 0.2.5
 BuildRequires:  rubygem(rspec)
-# Directory.find_data_file
-BuildRequires:  yast2 >= yast2-3.1.131
+# Directory.find_data_file >= yast2-3.1.131
+# cfg_mail.scr
+BuildRequires:  yast2 >= yast2-3.1.179
```

Comment:
Fixed. Thnx i didn't realize that there was a format error, just changed last digit.